tests/sqlparser_postgres.rs

```diff
@@ -774,51 +771,32 @@ fn parse_pg_regex_match_ops() {
 }
 
 #[test]
-fn parse_map_access_expr() {
```

Comment:
Could you please add back the the existing test parsing the simpler expression `SELECT foo[0] FROM foos` to make sure that that syntax is still supported (it would be fine if the resulting parse tree is slightly different)

I think it is important to ensure that this PR does not introduce a regression in terms of the syntax that is supported (I don't think it does, but I would like the test to ensure that this is the case)

Reply:
Ok. Thanks for the review!